### ISSUE TITLE
Exception for MESSAGE_REACTION_ADD packet

### ISSUE THREAD

**Issue by Bastian** [Javacord/Javacord#546]
```
2020-05-03 21:11:24,217 <WARN> <Javacord - Handlers Processor> <[]> <{shard=0}> <org.javacord.core.util.gateway.PacketHandler> Couldn't handle packet of type MESSAGE_REACTION_ADD. Please contact the developer! (packet: {"user_id":"702763150025556029","message_id":"706583461527420928","member":{"user":{"username":"nurophen","public_flags":64,"id":"702763150025556029","discriminator":"0001","avatar":"d98ebd26f98180e77deaeaabc4dce88f"},"roles":["238388552663171072"],"premium_since":null,"nick":null,"mute":false,"joined_at":"2020-04-26T09:17:40.787619+00:00","hoisted_role":null,"deaf":false},"emoji":{"name":"aqua_crying","id":"645829185167687681"},"channel_id":"381870553235193857","guild_id":"81384788765712384"})
java.lang.NullPointerException: null
	at org.javacord.core.entity.emoji.CustomEmojiImpl.<init>(CustomEmojiImpl.java:57) ~[classes/:?]
	at org.javacord.core.DiscordApiImpl.getKnownCustomEmojiOrCreateCustomEmoji(DiscordApiImpl.java:827) ~[classes/:?]
	at org.javacord.core.util.handler.message.reaction.MessageReactionAddHandler.lambda$handle$1(MessageReactionAddHandler.java:45) ~[classes/:?]
	at java.util.Optional.ifPresent(Optional.java:159) ~[?:1.8.0_231]
	at org.javacord.core.util.handler.message.reaction.MessageReactionAddHandler.handle(MessageReactionAddHandler.java:35) ~[classes/:?]
	at org.javacord.core.util.gateway.PacketHandler.lambda$handlePacket$0(PacketHandler.java:51) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_231]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_231]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_231]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_231]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_231]
```

**Comment by Bastian** [comment]
This is on the audio branch:
![image](https://user-images.githubusercontent.com/5033001/80923471-533e1b00-8d84-11ea-8ee8-e689235022b7.png)

**Comment by Bastian** [comment]
Was already fixed in v3.0.5.
My bad
